### PR TITLE
WebGL Shaders: Drawing fractals with JavaScript

### PR DESCRIPTION
**Pull request by NalinPlad**
Thanks for making a PR! Please fill in the below:

Name: Ajith

Do you run a club? Not a hack club at my school, only a robotics club

What relevant experiences do you have to help you write this Jam? A wrote [a version of this project](https://github.com/NalinPlad/gpu_fractals) using webgpu earlier this year. I also researched this topic and webgl / webgpu for long time outside of the project as well

Will you be available on Slack to provide support to Hack Clubbers who run into issues while going through your Jam? Yes

### CODE REVIEW

**Comment by Arpan-206** [comment]
This PR is missing a lot of the questions that haven't been answered.

Great Jam though!

**Comment by ymfpfp** [comment]
I think this is so so cool! (I've been wanting to learn about WebGL.) What's the timeframe for this though? Is it doable in one session?

**Comment by Arpan-206** [comment]
https://github.com/hackclub/Jams/pull/12/files

Should provide a pretty good idea of what kind of PR structure we're going for.

**Comment by NalinPlad** [comment]
> I think this is so so cool! (I've been wanting to learn about WebGL.) What's the timeframe for this though? Is it doable in one session?

I think the project can definitely be completed in an hour, if I go over most of the boilerplate code quickly. When I first did the project I spent most of the time on wikipedia trying to find resources on the actual math. So with the Jam, it would be much simpler and faster due to having a simple explanation of it created by me.

**Comment by NalinPlad** [comment]
> https://github.com/hackclub/Jams/pull/12/files
> 
> Should provide a pretty good idea of what kind of PR structure we're going for.

I updated the file, let me know if there are any other questions that you have for the Jam

**Comment by Arpan-206** [comment]
> > https://github.com/hackclub/Jams/pull/12/files
> > Should provide a pretty good idea of what kind of PR structure we're going for.
> 
> I updated the file, let me know if there are any other questions that you have for the Jam

Are you sure you pushed the changes?

**Review by sarthaktexas — APPROVED** [review]
I'm with JC, there's some time considerations that I'm concerned about but I trust your judgement.

**Review by sarthaktexas — CHANGES_REQUESTED** [review]
I seem to have missed the breakdown part, as well as some of the questions. Can you please add those?

**Comment by NalinPlad** [comment]
The Overview section was the breakdown, so I renamed it to that, and I added a rough meeting outline

File: jams/fractals.md [hunk 1]
```diff
@@ -19,19 +19,22 @@
 ---
 
 
-### Overview
+### Breakdown
```
Comment (Arpan-206) [inline jams/fractals.md:22]: Would you mind adding time estimates for each step?

**Review by Arpan-206 — CHANGES_REQUESTED** [review]
Minor changes

**Comment by NalinPlad** [comment]
Updates with time estimates and consolidated some of the steps

**Review by Arpan-206 — APPROVED** [review]
LGTM!

**Comment by NalinPlad** [comment]
@sarthaktexas any updates or any more questions?

**Review by linkai101 — APPROVED** [review]
LGTM too! Love how it makes a math concept actually fun to learn, and I can def see the appeal. I'm not familiar with WebGL but I look forward to seeing this jam!

**Review by sarthaktexas — APPROVED** [review]

**Comment by NalinPlad** [comment]
Thanks to all the reviewers! I look forward to working on this

**Comment by faisalsayed10** [comment]
Hi @NalinPlad, could you please provide me your slack username?

**Comment by NalinPlad** [comment]
> Hi @NalinPlad, could you please provide me your slack username?

@notmysql on slack

**Pull request merged**